press-release-faq.md

```diff
@@ -0,0 +1,176 @@
+relies on this restrictive processing for security. This model creates a constant tug of war between expanding the
+capability of BOS components and maintaining security. Importing code which expects to execute in a typical
+browser environment is not possible.
+
+In contrast, BWE leverages the sandboxing configuration of iframes built into browsers to isolate the execution
+of each component. Each component's code is passed to an invisible iframe along with an instance of Preact. Preact
+executes the components and the rendered output is serialized and passed out of the iframe. Then, the output of all
+the iframes is composed together into a single tree for render on the page. One way to think of this is that each
+component is run as its own isolated applet and given a messaging system to integrate with other components. This
+allows being significantly less restrictive with the code a component can run, since the only surface of exposure
+to the OWA is DOM nodes. It also means code written for a standard browser or Preact environment can be imported
+and run by components.
+
+## F.A.Q
+
+### General
+
+#### When will the previous VM stop being supported?
+
+The transition will occur in a few stages.
+
+There will be a beta period where both the VM and BWE are supported while we ensure BWE is stable and
+performant. During this period, we will encourage developers to build on BWE but breaking changes
+are more likely to occur as improvements to the engine are identified.
+
+Once we are confident in the production readiness of BWE, we will start a grace period during
+which the VM will continue to be supported while the community works to migrate existing BOS components to
+BWE. We will assist in this with ample guidance and hopefully some tooling to make the process easier.
+
+After the grace period, the VM will be deprecated and removed from the BOS runtime, since the security
+guarantees it provides are not sufficient for the future of BOS.
+
+### Building BOS Components with BWE
+
+#### What language and framework are BOS components written in?
+
+BOS components are written in TypeScript or JavaScript and use Preact as the rendering engine with the React compatibility layer enabled.
+One of the main goals of BWE is to have syntax that is minimally different from vanilla (p)react, and only stray from it where
+necessary to accomodate the architecture of the engine and sandbox communication patterns.
+
+See [components.md](components.md) for a detailed overview of the syntax required to build BOS components.
+
+#### What non-BOS code can I import?
+
+You can import npm packages via a CDN like https://esm.sh/
+
+You can expect for packages which are not dependent on direct access to the DOM or `window` object to work out of
+the box. Many packages which export custom React components are also supported due to our inclusion of preact/compat
+in the sandboxed environment.
+
+Packages which may not work:
```

Comment:
Maybe we should mention that this will not be supported by default but _could_ be supported in the future.

Reply:
yeah, intentionally leaving this a bit vague for right now since we don't know what's possible